Documentation/developer-guide/building-autoscoper.md

```diff
@@ -0,0 +1,69 @@
+# Building Autoscoper
```

Comment:
For now, let's reference the document using

For now, let's keep this document at the root and include it instead of copying it.

We will revisit when we update the instructions.

<pre>
```{include} ../../BUILD.md
```
</pre>